examples/cluster-dns/images/frontend/client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
ugggh, so what we really want is the boilerplate to complain if files get added with 2014....

Maybe the reason people get it 'wrong' is because they just cp the boilerplate into their header.  If we change the boilerplate to something really noticeable like "Copyright ***SET YEAR HERE*** The Kubernetes Authors All rights reserved" and instead of regex from 2014 -> 2015 we regex to "***SET YEAR HERE*** humans would be more likely to notice in the review?